src/components/WebexVideoSettings/WebexVideoSettings.scss

```diff
@@ -2,6 +2,10 @@
   display: flex;
   flex-direction: column;
 
+  .title {
+    font-family: $brand-font-medium;
```

Comment:
Do you need to specify `font-size: 1rem`??

Reply:
Sorry, I don't understand the question. It's not specified.